src/components/folders/FolderContent.jsx

```diff
@@ -56,47 +76,110 @@ const FolderContentItem = ({
         if (isFile) return dropdownItems
         return dropdownItems.filter(item => item.itemId !== 'move')
     }
-    return (
-        <Link className={`${contentStyles.component} ${contentStyles.card}`} to={link}>
-            <div type="button" className={`${elementStyles.card} ${contentStyles.card} ${elementStyles.folderItem}`}>
-                <div className={contentStyles.contentContainerFolderRow}>
-                    {
-                        isFile
-                        ? <i className={`bx bxs-file-blank ${elementStyles.folderItemIcon}`} />
-                        : <i className={`bx bxs-folder ${elementStyles.folderItemIcon}`} />
+
+    const FolderItemContent = (
+        <div type="button" className={`${elementStyles.card} ${contentStyles.card} ${elementStyles.folderItem}`}>
+            <div className={contentStyles.contentContainerFolderRow}>
+                {
+                    isFile
+                    ? <i className={`bx bxs-file-blank ${elementStyles.folderItemIcon}`} />
+                    : <i className={`bx bxs-folder ${elementStyles.folderItemIcon}`} />
+                }
+                <span className={`${elementStyles.folderItemText} mr-auto`} >{deslugifyPage(title)}</span>
+                {
+                    numItems
+                    ? <span className={`${elementStyles.folderItemText} mr-5`}>{numItems} item{numItems === 1 ? '' : 's'}</span>
+                    : null
```

Comment:
Could we check for `numItems !== null` instead, so that folders with 0 items still have the label show up?

Reply:
46ee954